tests/unit/test_modelling.py

```diff
@@ -305,6 +305,14 @@ def test_input_error(self):
         with self.assertRaisesRegex(TypeError, exact(msg)):
             TrainingDatum(1, 1)
 
+    def test_empty_input_error(self):
+        """Test that a ValueError is raised if the constructor arg 'input'
+        is an empty Input."""
+
+        msg = "Argument 'input' is empty, it must contain a value."
```

Comment:
We changed the message but not in the test 🤦

Reply:
Oopsies 🤦🏼